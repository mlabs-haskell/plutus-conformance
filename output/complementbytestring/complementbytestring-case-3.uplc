[ (builtin complementByteString) (con bytestring #b00b) ]