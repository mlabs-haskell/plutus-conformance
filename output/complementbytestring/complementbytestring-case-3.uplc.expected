(con bytestring #4ff4)